frontend/src/tests/components/common/UploadedFilesList.spec.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { UploadedFilesList } from "@/components/common/UploadedFilesList";
+import * as MOCK from "@/tests/mocks/dataMocks";
+
+const props = {};
+const dispatchProps = {};
+
+const setupProps = () => {
+  props.selectedDocId =
+    MOCK.MINES.mines[MOCK.MINES.mineIds[0]].mine_expected_documents[0].exp_document_guid;
+  props.mineId = MOCK.MINES.mineIds[0];
+  props.expectedDocumentStatusOptions = MOCK.EXPECTED_DOCUMENT_STATUS_OPTIONS.options;
+  props.mines = MOCK.MINES.mines;
+};
```

Comment:
If these props are required in the component, should they not be required by the tests?
removeMineDocumentFromExpectedDocument
fetchMineRecordById
expectedDocumentStatusOptions

Reply:
All of these should be mocked appropriately in the test and I don't believe you can require props in tests.

Reply:
I had just meant that the test should fail if not all required props are provided. I didn't notice that a couple of them were set up in `setupDispatchProps`